tests/fixtures/functions/function_Mixed_Name.sql

```diff
@@ -13,3 +13,25 @@ RETURNS TABLE("mVt" bytea, key text) AS $$
       WHERE "Geom" && ST_Transform(ST_TileEnvelope("Z", x, y), 4326)
   ) as tile WHERE geom IS NOT NULL) src
 $$ LANGUAGE SQL IMMUTABLE STRICT PARALLEL SAFE;
+
+COMMENT ON FUNCTION "MixedCase"."function_Mixed_Name" (int4,INT4,INT4) IS $tilejson$
```

Comment:
See if you can modify these somehow to auto-verified with `'{...}'::json` approach instead of a unverified string

Reply:
Weired. PostgreSQL complained syntax error. Need a little bit research on.
```SQL
COMMENT ON FUNCTION osm_trees(INTEGER,INTEGER,INTEGER) IS ('{"a":1}'::json::TEXT)
```

Reply:
Seems it only receive literal value?

Reply:
@sharkAndshark use this method:

```sql
DO $do$ BEGIN
    EXECUTE 'COMMENT ON FUNCTION public.function_null (INT4,INT4,INT4) IS $tj$' || $$
    {
      "minzoom": 0,
      "maxzoom": 18,
      "bounds": [-180, -85, 180, 85],
      "vector_layers": []
    }
    $$::json || '$tj$';
END $do$;
```